OpenRA.Mods.Common/Widgets/Logic/Lobby/LobbyUtils.cs

```diff
@@ -104,6 +104,18 @@ public SlotDropDownOption(string title, string order, Func<bool> selected)
 			dropdown.ShowDropDown("SPAWN_DROPDOWN_TEMPLATE", 150, spawnPoints, setupItem);
 		}
 
+		/// <summary>Splits a string into two parts on the first instance of a given token.</summary>
+		static Pair<string, string> SplitOnFirstToken(string input, string token = "\\n")
+		{
+			if (string.IsNullOrEmpty(input))
```

Comment:
Ok, though not clear to me. I guess not specifying Description means null. It isn't possible to specify "" in yaml so it would have an effect only when used from code. anyway ok for me to keep it, is surely simpler.